### PR TITLE
Categories page

### PR DESCRIPTION
**Pull request by i-am-a-codegirl**
Please review the category page of my quiz site

It has the following features

nav : links to categories and rules page
main: with introduction part and different categories card along with Call to action button in the cards
Footer: links to the social websites

image

image

image

image

### CODE REVIEW

File: categories/categories.css [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+    font-family: 'Inter', sans-serif;
+    font-family: 'Mukta', sans-serif;
+    background-image: url("blue-copy-space-digital-background_23-2148821698.webp");
+    background-repeat: no-repeat;
+    background-size: cover;
+   
+box-sizing: border-box;
+margin: 0px;
+padding: 0px;
+/* background-color: #e056fd;
```
Comment (pnchinmay) [inline categories/categories.css]: Remove these comments

File: categories/categories.css [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+nav h2{
+    position: relative;
+  
+    font-size: 4rem;
+    line-height: 1.2rem;
+    letter-spacing: 0.125rem;
+}
+
+.imp-text{
+    font-family: 'Inter', sans-serif;
```
Comment (pnchinmay) [inline categories/categories.css]: Keep the indentation same at all places

File: categories/categories.css [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+
+  .card-content {
+    margin: 0.5rem;
+  }
+
+
+
+  .card {
+      color: var(--off-white);
+      background-color: #2a2a72;
```
Comment (pnchinmay) [inline categories/categories.css]: Change it to some color variable

**Review by pnchinmay — APPROVED** [review]
Left a few comments
Everything else looks good :100:

File: categories/categories.css [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+    width: 100vw;
+    height: 16vw;
+    grid-area: nav;
+    
+
+}
+
+nav h2{
+    position: relative;
+
```
Comment (khushijohri2001) [inline categories/categories.css]: Remove this extra line here

File: categories/categories.css [hunk 5]
```diff
@@ -0,0 +1,345 @@
...
+}
+
+.current-pg{
+    color: var(--secondary-color);
+}
+
+
+
+nav ul{
+    margin-left: auto;
```
Comment (khushijohri2001) [inline categories/categories.css]: Keep the indentation same everywhere. Ideally, 2 spaces in HTML/CSS according to Google Style Guide

**Review by khushijohri2001 — APPROVED** [review]

**Pull request merged**